pkg/codegen/java/gen.go

```diff
@@ -1604,31 +1596,29 @@ func (mod *modContext) genFunctions(ctx *classFileContext, addClass addClassMeth
 		// Emit the args and result types, if any.
 		if fun.Inputs != nil {
 			// Generate "{Function}Args" class for invokes that return Output<T>
+			// Notice here using fun.Inputs.InputShape.Properties which use the shape which accepting Outputs
```

Comment:
👍 

There's a bit of misnomer here. In other languages there is an `Input<T>` type, canonically (in TypeScript) it's an untagged union `T | Output<T>`. In Java we made the controversial decision to not model that type at all, so we only have `Output<T>` - but the cross-language Go framework is still calling that `InputShape` - which is fine. at the meta-level `InputShape` will inject `InputType` wrappers, that will be translated to `Output<T>` during the Java specific codegen pass.